project-koku/koku
# [COST-2764] Add raw_currency to OCP tables

## Jira Ticket

[COST-2764](https://issues.redhat.com/browse/COST-2764)

## Description
Add raw_currency to OCP tables for OCP on cloud 

## Testing

1. Checkout Branch
2. Restart Koku
3. Run the following commands: 
```
make create-test-customer
make load-test-customer-data 
```
4. Verify that `raw_currency` is in the following tables using PGSQL: 
* reporting_ocpusagelineitem_daily_summary
* reporting_ocp_cost_summary_by_node_p
* reporting_ocp_pod_summary_by_project_p
* reporting_ocp_cost_summary_p
* reporting_ocp_pod_summary_by_project_p
* reporting_ocp_pod_summary_p
* reporting_ocp_volume_summary_by_project_p
* reporting_ocp_volume_summary_p
```
PGPASSWORD=postgres psql postgres -U postgres -h localhost -p 15432 
postgres=# \x
postgres=# set search_path=acct10001;
postgres=# select * from {TABLES_ABOVE};

```
## Notes

...
